docs/__genrule_common.soy

```diff
@@ -60,6 +60,14 @@
       <dd>Expands to the location of the output of the build rule. This
       means that you can refer to these without needing to be aware of how
       Buck is storing data on the disk mid-build.</dd>
+
+      <dt><code>$(maven_coords /&#x2F;path/to:target)</code></dt>
+      <dd>Expands to the Maven coordinates for the build rule. This allows
+      you to access the Maven coordinates for Maven-aware build rules. The
+      format of the expansion is: <code>
+      &lt;groupId&gt;:&lt;artifactId&gt;[:&lt;extension&gt;[:&lt;classifier&gt;]]:&lt;version&gt;
```

Comment:
It seems like the `{literal}` tag blocks HTML escaping, and that `<` and `>` are not auto-escaped within the HTML template without `{literal}`. However, we can make things more readable with a `{let}` block, which should render/escape the angle brackets on replacement. I will push an update.